test-data/unit/check-except-star.test

```diff
@@ -0,0 +1,36 @@
+[case testExceptStarForbiddenBasic]
+# flags: --python-version 3.11
+[builtins fixtures/exception.pyi]
```

Comment:
Hmm, I'm pretty sure fixtures need to go after the test body. I think putting the fixture here means that the test body isn't being run.